homework/Lesson 2 challenge-Andre.py

```diff
@@ -0,0 +1,28 @@
+name = input("what is your name?: ")
+print("please answer the following with numbers only:")
+pizzas1 = input("How many pizzas would you like? ")
+pepperonis1 = input("How many pepperonis do you want? ")
+olives1 = input("How many olives do you want? ")
+print(str(name) + " ordered " + str(pizzas1) + " pizzas with " + str(pepperonis1) + " pepperonis and " + str(olives1) + " olives.")
+print(" ")
+print(" ")
+print(" ")
+print("second order")
+print("please answer the following with numbers only:")
+pizzas2 = input("How many pizzas would you like? ")
+pepperonis2 = input("How many pepperonis do you want? ")
+olives2 = input("How many olives do you want? ") 
+print(str(name) + "'s second order is " + str(pizzas2) + " pizzas with " + str(pepperonis2) + " pepperonis and " + str(olives2) + " olives.")
+print(" ")
+print(" ")
+print(" ")
+print("third order")
+print("please answer the following with numbers only:")
+pizzas3 = input("How many pizzas would you like? ")
+pepperonis3 = input("How many pepperonis do you want? ")
+olives3 = input("How many olives do you want? ")
+print(str(name) + "'s third order is " + str(pizzas3) + " pizzas with " + str(pepperonis3) + " pepperonis and " + str(olives3) + " olives.")
+average = print(str(name) + ''' ordered an average of:
+''' + str(float((int(pizzas1) + int(pizzas2) + int(pizzas3)) / 3)) + ''' pizzas with
+''' + str(float((int(pepperonis1) + int(pepperonis2) + int(pepperonis3)) / 3)) + ''' pepperonis and
+''' + str(float((int(olives1) + int(olives2) + int(olives3)) / 3)) + " olives per order.")
```

Comment:
`'''` is only needed on the first and last line of the printed things, but this works too

Reply:
Tom, you are too fast. When I got in, it seems I don't need to do anyting:)

Reply:
lol